<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgConceptsHistorical</class>
 <widget class="QDialog" name="wdgConceptsHistorical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historical concepts report</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/history.png</normaloff>:/xulpymoney/history.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="widget">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/hucha.png</normaloff>:/xulpymoney/hucha.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Historical report</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="mqtw" name="mqtwReport" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionShowMonth">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/search.png</normaloff>:/xulpymoney/search.png</iconset>
   </property>
   <property name="text">
    <string>Show month operations</string>
   </property>
   <property name="toolTip">
    <string>Show month operations</string>
   </property>
  </action>
  <action name="actionShowYear">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/document-preview-archive.png</normaloff>:/xulpymoney/document-preview-archive.png</iconset>
   </property>
   <property name="text">
    <string>Show year operations</string>
   </property>
   <property name="toolTip">
    <string>Show year operations</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtw</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
